src/ApparentHorizons/StrahlkorperGr.cpp

```diff
@@ -615,6 +614,39 @@ double dimensionful_spin_magnitude(
 
   return get_spin_magnitude(potentials, spin_function, area_element, ylm);
 }
+
+template <typename Frame>
+std::array<double, 3> spin_vector(const double spin_magnitude,
+                                  const Scalar<DataVector>& area_element,
+                                  const DataVector& radius,
+                                  const tnsr::i<DataVector, 3, Frame>& r_hat,
+                                  const Scalar<DataVector>& ricci_scalar,
+                                  const Scalar<DataVector>& spin_function,
+                                  const YlmSpherepack& ylm) noexcept {
+  std::array<double, 3> spin_vector = {{0.0, 0.0, 0.0}};
+  auto integrand = make_with_value<Scalar<DataVector>>(radius, 0.0);
+  for (size_t i = 0; i < 3; ++i) {
+    // Make the relative coordinates be relative to a center such that
+    // the mass dipole moment vanishes.
```

Comment:
I think "Compute a coordinate center such that the mass dipole moment vanishes" is clearer, if it is still accurate.